README.md

```diff
@@ -4,10 +4,12 @@ This repo has a "Hello World" Go application that uses the [pgx driver](https://
 
 To run the code:
 
-1. Start a [local, insecure CockroachDB cluster](https://www.cockroachlabs.com/docs/stable/start-a-local-cluster.html).
+1. Start a [demo CockroachDB cluster](https://www.cockroachlabs.com/docs/stable/cockroach-demo.html).
 
-2. Create a `bank` database and `maxroach` user as described in [Build a Go app with CockroachDB](https://www.cockroachlabs.com/docs/stable/build-a-go-app-with-cockroachdb.html#insecure).
+1. Create a `bank` database and a user and password as described in [Build a Go app with CockroachDB](https://www.cockroachlabs.com/docs/stable/build-a-go-app-with-cockroachdb.html).
 
-3. From the [SQL client](https://www.cockroachlabs.com/docs/stable/cockroach-sql.html): `GRANT ALL ON DATABASE bank TO maxroach`
+1. From the SQL client: `GRANT ALL ON DATABASE bank TO maxroach`
```

Comment:
Either change previous step to create user `maxroach` or change the `GRANT` statement to use a user variable.

Reply:
Done.